airbyte-cdk/python/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apk --no-cache upgrade \
     && apk --no-cache add tzdata build-base
 
 # install airbyte-cdk
-RUN pip install --prefix=/install airbyte-cdk==0.73.0
+RUN pip install --prefix=/install airbyte-cdk==0.73.1
```

Comment:
no need to bump the version. this is done automatically by the publish github action

Reply:
Fixed